src/layout/strategy/strategy.js

```diff
@@ -0,0 +1,5 @@
+import withinContainer from './withinContainer';
+
+export default {
+    withinContainer: withinContainer
```

Comment:
Does that mean noop is now equivalent to fc.util.fn.identity?

On Thu, 19 Nov 2015 at 16:37 Colin Eberhardt <notifications@github.com>
wrote:

> In src/layout/strategy/strategy.js
> <https://github.com/ScottLogic/d3fc/pull/775#discussion_r45364787>:
>
> > @@ -0,0 +1,5 @@
> > +import withinContainer from './withinContainer';
> > +
> > +export default {
> > +    withinContainer: withinContainer
>
> Added - and removed containerWidth / height from noop.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ScottLogic/d3fc/pull/775/files#r45364787>.
>